js_modules/dagster-ui/packages/ui-core/src/nav/LeftNavItem.tsx

```diff
@@ -21,8 +27,31 @@ export const LeftNavItem = React.forwardRef(
     const {label, leftIcon, path, repoAddress, schedules, sensors} = item;
 
     const [showDialog, setShowDialog] = React.useState(false);
+    const repositoryId = useRepository(repoAddress)?.repository.id;
 
-    const rightIcon = () => {
+    useQuery<InstigationStatesQuery, InstigationStatesQueryVariables>(INSTIGATION_STATES_QUERY, {
+      variables: {
+        repositoryID: repositoryId!,
```

Comment:
This value can definitely be `undefined` here though, right?

Reply:
ah yeah I'll skip until its available in this case (it should be available once the WorkspaceContext is populated)

Reply:
Yes, but I'm skipping on line 36 in that case.

Reply:
so either I use an expect-error or do this.